bin/dev-requirements

```diff
@@ -334,44 +323,32 @@ main() {
     required_clojure_version=$(get_required_clojure_version)
 
     echo -e "  ${BOLD}Required Tools:${NC}"
-    echo "    • Node.js ≥22 (currently specified in .nvmrc)"
-    echo "    • Yarn Classic 1.x (NOT Yarn 2+, NO Corepack)"
-    echo "    • Java 21 with Eclipse Temurin distribution (for Clojure backend)"
+    echo "    - Node.js >= 22 (currently specified in .nvmrc)"
+    echo "    - Yarn Classic 1.x (NOT Yarn 2+, NO Corepack)"
+    echo "    - Java 21 with Eclipse Temurin distribution (for Clojure backend)"
     if [ -n "$required_clojure_version" ]; then
-        echo "    • Clojure CLI (for Clojure $required_clojure_version specified in deps.edn)"
+        echo "    - Clojure CLI (for Clojure $required_clojure_version specified in deps.edn)"
     else
-        echo "    • Clojure CLI"
+        echo "    - Clojure CLI"
     fi
-    echo "    • Babashka (for mage task automation)"
+    echo "    - Babashka (for mage task automation)"
```

Comment:
it's semantics, but you don't need to install babashka to use mage. As it is now, the `:min-bb-version` of bb (from bb.edn) is auto-installed into `./bin/bb` and only that version is used by mage, not what is on people's path.